thinc/tests/test_util.py

```diff
@@ -39,11 +47,23 @@ def test_get_width_fail(obj):
         get_width(obj)
 
 
-def test_array_module_cpu_gpu_helpers():
-    xp = get_array_module(0)
-    assert hasattr(xp, "ndarray")
-    assert is_numpy_array(numpy.zeros((1, 2)))
-    assert not is_numpy_array((1, 2))
+@pytest.mark.parametrize("xp", ALL_XP)
+def test_array_module_cpu_gpu_helpers(xp):
+    error = ("Only numpy and cupy arrays are supported"
+             ", but found <class 'int'> instead. If "
+             "get_array_module module wasn't called "
+             "directly, this might mean a bug in Thinc.")
```

Comment:
```suggestion
             "directly, this might indicate a bug in Thinc.")
```

Again, I think we shouldn't have complete error messages copied in the tests or it's too hard to maintain.